src/commands/bundle.zsh

```diff
@@ -9,7 +9,7 @@ antigen-bundle () {
     return 1
   fi
 
-  builtin typeset -A bundle; -antigen-parse-args 'bundle' ${=@}
+  builtin typeset -A bundle; -antigen-parse-args 'bundle' $@
```

Comment:
Sorry for the slow responses.

`antigen theme desyncr/void void`. But I believe any theme should fail to load, the tricky part is it doesn't fail in tests.

I'm gonna try to give it a look today. Let me know if you found anything.